test/validate_test/email_test.exs

```diff
@@ -15,6 +15,11 @@ defmodule Fields.ValidateEmailTest do
     StreamData.string(:alphanumeric, min_length: 1, max_length: 63)
   end
 
+  test "local part of email containing only numbers is valid"  do
```

Comment:
@RobStallion when there is a specific test like this that others won't have a *clue* about, 
consider adding the issue number e.g: #39 😉